packages/experimental/src/global/js/package-settings.js

```diff
@@ -0,0 +1,82 @@
+//
```

Comment:
This file shouldn't exist anymore. It is located in `packages/cloud-cognitive/global/js/package-settings.js` now